docs/installconfig/installation/postinstallation.md

```diff
@@ -1,19 +1,19 @@
 ---
 layout: default
-title: Post installation tasks
+title: Post Installation Tasks
 parent: Installing & configuring
 nav_order: 6
 ---
 
-# Post installation tasks
+# Post Installation Tasks
 
-// TODO: [PJAN-1310] Make this shine
+This chapter describes tasks you can perform after installing an instance of KEEP.
 
-## [securing ports](../configuration/configuringPorts)
+## [Securing ports](../configuration/configuringPorts)
 
-## [enable database](../../usingkeep/enablingadb)
+## [Enable a database](../../usingkeep/enablingadb)
 
-## [configure jwt](../configuration/security/authentication)
+## [Configure JWT](../configuration/security/authentication)
```

Comment:
Please change to "Configuring JWT" for consistency (little something I learned when working with documentation experts)

Reply:
A sentence per link, to summarise why this is recommended, necessary or important would make a lot of sense